packages/paste-core/components/alert-dialog/src/index.tsx

```diff
@@ -48,6 +49,7 @@ export const AlertDialog = React.forwardRef<HTMLDivElement, AlertDialogProps>(
       onConfirmLabel,
       onDismiss,
       onDismissLabel,
+      disableDestructive,
```

Comment:
question/chore: needs to be added to the props table, yeah?

Reply:
yes good catch